### DISCUSSION TITLE
How to hide page numbers in favorites titles?

### DISCUSSION THREAD

**Discussion by nijazm** [sumatrapdfreader/sumatrapdf#3487] [Documentation]
When I add some pdf document to favorites, it shows file name and page number.
But I don't need to know page numbers and some numbers look ugly in form of weird symbols instead of numbers.
So how to hide page numbers in favorites titles, and only display file names, without ":" and anything after it (page numbers)?

**Comment by GitHubRulesOK** [comment]
Most favourites would simply be page 0 but PDF page 0 can be anything (normally Cover or 1 or i or A etc.) The point is that the file needs a designated target filename to open

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/af2a5640-20cf-4972-8a98-ebf276dd0e4a)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/9f6896b5-e25b-46ad-be78-142b5cc86a51)
